contrib/eventcmd-examples/eventcmd.sh

```diff
@@ -11,7 +11,7 @@ case "$1" in
 #	songstart)
 #		echo 'naughty.notify({title = "pianobar", text = "Now playing: ' "$title" ' by ' "$artist" '"})' | awesome-client -
 
-#		echo "$title -- $artist" > $HOME/.config/pianobar/nowplaying
+#		echo "$title -- $artist" > $XDG_CONFIG_HOME/pianobar/nowplaying
```

Comment:
If so, why not , 

    echo "${title} -- ${artist}" > "${XDG_CONFIG_HOME}/pianobar/nowplaying"